File: 1000/1000.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        String input = scanner.nextLine();

        String[] split = input.split(" ");
        int A = Integer.parseInt(split[0]);
        int B = Integer.parseInt(split[1]);

        System.out.println(A + B);
        scanner.close();
    }
}

File: 1000/1000.js
import * as readline from 'node:readline/promises'
import { stdin , stdout } from 'node:process'

const rl = readline.createInterface({ input:stdin, output:stdout })

let input = await readLine();
input = input.split(' ');

let A = parseInt(input[0]);
let B = parseInt(input[1]);

console.log(A + B);

rl.close();

async function readLine(){
    return new Promise(resolve => {
        rl.once('line', line => {
            resolve(line);
        })
    });
}